docs/basic-docs/atomicdex-api-20-dev/withdraw_init.md

```diff
@@ -0,0 +1,63 @@
+# withdraw_init
+
+The `withdraw_init` method generates, signs, and returns a transaction that transfers the `amount` of `coin` to the address indicated in the `to` argument. It is only used for z coins like ZOMBIE, which may take some time to complete. The status of this method can be queried via the [withdraw_status](withdraw_status.html) method, or cancelled with [withdraw_status](withdraw_cancel.html).
+
+
+### Arguments
+
+| Structure     | Type             | Description                                                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| coin          | string           | the name of the coin the user desires to withdraw                                                                                         |
+| to            | string           | coins are withdrawn to this address                                                                                                       |
+| amount        | string (numeric) | the amount the user desires to withdraw, ignored when `max=true`                                                                          |
+| max           | bool             | withdraw the maximum available amount                                                                                                     |
+| fee.type      | string           | type of transaction fee; possible values: `UtxoFixed` or `UtxoPerKbyte`                                                                   |
+| fee.amount    | string (numeric) | fee amount in coin units, used only when type is `UtxoFixed` (fixed amount not depending on tx size) or `UtxoPerKbyte` (amount per Kbyte) |
```

Comment:
We also have arguments `memo` (optional and should be used carefully, as Artem suggested), and [`from`](https://github.com/KomodoPlatform/developer-docs/pull/403/files#r1058952296) (optional). As I can see, we use `from` for `WithdrawRequest` in trezor integration, in the case of Iguana there will be an error `'from' is not supported if the coin is initialized with an Iguana private key`

Reply:
> Ain't 100% sure, but `memo` sounds like tendermint-only thing here.

We merged the PR related to memo use in ARRR withdraw https://github.com/KomodoPlatform/atomicDEX-API/pull/1574

Reply:
Memo tested with ARR successfully (tho not seen in tx_history output as `"error_type":"NotSupportedFor","error_data":"ARRR"`)
Added to the doc along with example to satisfy https://github.com/KomodoPlatform/atomicDEX-API/issues/1589#event-8169743944

Reply:
> tho not seen in tx_history output as `"error_type":"NotSupportedFor","error_data":"ARRR"`

Yep, thats correct, `my_tx_history_v2_rpc` now works only for Bch, SlpToken, UtxoCoin, QtumCoin, Tendermint and TendermintToken